lib/options.js

```diff
@@ -12,90 +12,104 @@ module.exports = {
    */
-  getOptions: function(mode = 'document') {
+  getOptions: function(mode = 'document', criteria = {}) {
+    // Override mode & criteria if first arg is criteria (objects)
+    if (typeof mode === 'object') {
+      criteria = mode;
+      mode = 'document';
+    }
+
     let optsArray = [
       {
-        name: 'Set request name source',
+        name: 'Naming requests',
         id: 'requestNameSource',
         type: 'enum',
-        default: 'fallback',
-        availableOptions: ['url', 'fallback'],
-        description: 'Option for setting source for a request name',
-        external: true
+        default: 'Fallback',
+        availableOptions: ['Url', 'Fallback'],
+        description: 'Determines how the requests inside the generated collection will be named.' +
+        ' If “Fallback” is selected, the request will be named after one of the following schema' +
+        ' values: description, operationid, url.',
+        external: true,
+        usage: ['CONVERSION']
       },
       {
         name: 'Set indent character',
         id: 'indentCharacter',
         type: 'enum',
-        default: ' ',
-        availableOptions: [' ', '\t'],
+        default: 'Space',
+        availableOptions: ['Space', 'Tab'],
         description: 'Option for setting indentation character',
-        external: true
+        external: true,
+        usage: ['CONVERSION']
       },
       {
-        name: 'Toggle for collapsing folder for long routes',
+        name: 'Collapse redundant folders',
         id: 'collapseFolders',
         type: 'boolean',
         default: true,
-        description: 'Determines whether the importer should attempt to collapse redundant folders into one.' +
-         'Folders are redundant if they have only one child element, and don\'t' +
-         'have any folder-level data to persist.',
-        external: true
+        description: 'Importing will collapse all folders that have only one child element and lack ' +
+        'persistent folder-level data.',
+        external: true,
+        usage: ['CONVERSION']
       },
       {
-        name: 'Set root request parameters type',
+        name: 'Request parameter generation',
         id: 'requestParametersResolution',
         type: 'enum',
-        default: 'schema',
-        availableOptions: ['example', 'schema'],
-        description: 'Determines how request parameters (query parameters, path parameters, headers,' +
-         'or the request body) should be generated. Setting this to schema will cause the importer to' +
-         'use the parameter\'s schema as an indicator; `example` will cause the example (if provided)' +
-         'to be picked up.',
-        external: true
+        default: 'Schema',
+        availableOptions: ['Example', 'Schema'],
+        description: 'Select whether to generate the request parameters based on the schema or the' +
+        ' examples in the schema.',
```

Comment:
Can you add markdown supported link back to schema and example objects? I did add it in previous commit.